src/Language/Docker/Parser/Run.hs

```diff
@@ -29,7 +28,8 @@ data RunMountArg
   | MountArgSource SourcePath
   | MountArgTarget TargetPath
   | MountArgType Text
-  | MountArgUid Integer
+  | MountArgUid Text
```

Comment:
I wonder if this should be a sum type with either an int or a text. What do you think?

Reply:
Maybe. But that would bring some complexity for the consumers of this library when dealing with these values, because then they can be either an `Integer` or a `Text`. This way it also is somewhat consistent with the mode flag, and the `--chown` flags from the `COPY` instruction. Those take their arguments as `Text` as well.
Since there is not much arithmetic to be expected with these values, I think `Text` will do just fine, or am I missing an advantage of making this a sum type?